ApiReview/API.Polly.Core/NoDocs/Polly/ResilienceStrategyOptions.cs

```diff
@@ -0,0 +1,12 @@
+// Assembly 'Polly.Core'
+
+using System.Runtime.CompilerServices;
+
+namespace Polly;
+
+public abstract class ResilienceStrategyOptions
+{
+    public string? StrategyName { get; set; }
```

Comment:
Should each strategy add some default name?

Pros: can be helpful for diagnostics (???)

Cons: redundant information unless someone really specifies custom name?

Reply:
I'd use as default name the "Type" of policy. i.e. "Retry", "Timeout"

Reply:
Fixed by #1475